ui/tabs.js

```diff
@@ -713,7 +713,7 @@ $.widget( "ui.tabs", {
 	_getIndex: function( index ) {
 		// meta-function to give users option to provide a href string instead of a numerical index.
 		if ( typeof index === "string" ) {
-			index = this.anchors.index( this.anchors.filter( "[href$='" + index + "']" ) );
+			index = this.anchors.index( this.anchors.filter( "[href$='" + $.ui.escapeId( index ) + "']" ) );
```

Comment:
Ah right sorry we use urls as a form of id in a similar case in mobile. Was thinking of that. Either way though yes this does work here to sanitize this for a selector. We use this in mobile as part of our navigation as we need to frequently sanitize things for use as a selector.